cmake/defaults/Version.cmake

```diff
@@ -22,6 +22,6 @@
 # language governing permissions and limitations under the Apache License.
 #
 # Versioning information
-set(USD_ARNOLD_MAJOR_VERSION "0")
-set(USD_ARNOLD_MINOR_VERSION "0")
-set(USD_ARNOLD_PATCH_VERSION "3")
+set(HD_RPR_MAJOR_VERSION "0")
```

Comment:
Versioning should be different for houdini build and RPR build you think?  Or should be 18.0.0 for Houdini 18 build?

Reply:
Potentially, hdRpr can be used in any software, so I don’t think that binding version to any particular soft like Houdini is a good idea. And having separate versioning seems like an additional headache